docs/_data/walkthrough_data.yml

```diff
@@ -0,0 +1,308 @@
+# STANDARD DATA: SAM/BAM/CRAM files following the specs
```

Comment:
Please, check this file for possible errors in the definitions/names and tell me if a new file will be required.